assets/prototype/domain/shared/predicates/shared/selectionPredicates/index.test.ts

```diff
@@ -0,0 +1,143 @@
+/**
+ * External dependencies
+ */
+import { omit } from 'ramda';
+
+/**
+ * Internal dependencies
+ */
+import {
+	entityDbId,
+	entityGuId,
+	entityHasDbId,
+	entityHasGuid,
+	findEntityByDbId,
+	findEntityByGuid,
+	entitiesWithDbIdInArray,
+	entitiesWithGuIdInArray,
+} from './index';
+import { nodes as datetimes } from '../../../../../domain/eventEditor/data/queries/datetimes/test/data';
+import { nodes as tickets } from '../../../../../domain/eventEditor/data/queries/tickets/test/data';
```

Comment:
It feels so good to use centralized test data 👍

Reply:
To make things even better we can add under `domain/eventEditor/data/queries` a `test/data.ts` file and re-export dates, tickets, etc, so we will end up with
`import { datetimes, tickets ... } from '../../../../../domain/eventEditor/data/queries/test/data';`